BRLS/src/library/borrowitem/BorrowItemUI.java

```diff
@@ -4,24 +4,24 @@
 
 public class BorrowItemUI {
 	
-	public static enum uI_STaTe { INITIALISED, READY, RESTRICTED, SCANNING, IDENTIFIED, FINALISING, COMPLETED, CANCELLED };
+	public static enum BorrowItemUIState { INITIALISED, READY, RESTRICTED, SCANNING, IDENTIFIED, FINALISING, COMPLETED, CANCELLED };
```

Comment:
Spaces at the start of the enum brackets and the end, please fix it. Same with the other class BorrowItemControl

Reply:
I have rectified this mistake

Reply:
Thank you for removing the spaces